release-notes/azure/v11.2.0.md

```diff
@@ -0,0 +1,69 @@
+## :zap: Giant Swarm Release 11.2.0 for Azure is now active for you! :zap:
+
+This release upgrades the NGINX Ingress Controller app to upstream ingress-nginx v0.29.0. Among multiple improvements this upgrade comes with a **breaking change** - default SSL ciphers no longer include AES-CBC based ciphers since they are considered weak. As side-effect this also drops SSL support for old browsers and clients which do not support AES-GCM ciphers (e.g. Safari 9). At your own risk weak ciphers can still be enabled on demand independently for each cluster.
```

Comment:
```suggestion
This release upgrades the NGINX Ingress Controller app to upstream ingress-nginx v0.29.0. Among multiple improvements, this upgrade comes with a **breaking change** - default SSL ciphers no longer include AES-CBC based ciphers since they are considered weak. As a side effect, this also drops SSL support for old browsers and clients which do not support AES-GCM ciphers (e.g. Safari 9). At your own risk, weak ciphers can still be enabled on demand independently for each cluster.
```